homeassistant/components/steam_online/sensor.py

```diff
@@ -140,10 +147,28 @@ def _get_current_game(self):
         _LOGGER.error("Unable to find name of app with ID=%s", game_id)
         return repr(game_id)
 
+    def _get_last_online(self):
+        last_online = datetime.fromtimestamp(
```

Comment:
Will this return UTC time?

We have this util:
https://github.com/home-assistant/home-assistant/blob/2c1a7a54cdae4619ab6f194af1f5554717f6ec2b/homeassistant/util/dt.py#L96

Times in state attributes must be absolute UTC.

Reply:
Thanks for the suggestion, changed it to use dt and set to UTC